.rhcicd/clowdapp-engine.yaml

```diff
@@ -217,6 +217,10 @@ parameters:
 - name: CPU_REQUEST
   description: CPU request
   value: 500m
+- name: ENV_BASE_URL
+  default: https://console.redhat.com
+  description: The environment's base URL
+  required: true
```

Comment:
This is not needed since we have a default value in the Java code.
```suggestion
```

Reply:
True! Removed!